src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1376,11 +1379,17 @@ From there, simply click on a Markdown file or SVG icon in the File Explorer and
   <data name="Delete_Dialog_Description" xml:space="preserve">
     <value>Are you sure you want to delete this item?</value>
```

Comment:
```suggestion
     <value>Are you sure you want to delete the item {itemToDelete}?</value>
```

@niels9001 
Can we add the name of the size to the dialog like suggested above or like this:
```
Are you sure you want to delete this item?
  <Name of size>

[yes] [no]
```

Reply:
might as well strip the "..this item.." too then?

Reply:
Can you give a complete example?

Reply:
"Are you sure you want to delete '{itemToDelete}'?"